docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -242,3 +316,30 @@ In order to avoid a conflict between multiple Elasticsearch clusters writing the
 - sets the `base_path` to `snapshots/<namespace>-<esName>` when it is not provided, for Azure, GCS and S3 repositories
 - appends `<namespace>-<esName>` to `location` for a FS repository
 - appends `<namespace>-<esName>` to `path` for an HDFS repository
+
+[float]
+[id="{p}-{page_id}-specifics-secret-mounts"]
+== Specifics for secret mounts
+
+In ECK `2.11.0` a new field was introduced to the Elastic Stack configuration policy under the `elasticsearch` field called `secretMounts`.
+This field allows users to specify a user created secret and a mountPath to indicate where this secret should be mounted in the Elasticsearch Pods that are managed by the Elastic Stack configuration policy.
+This field can be used to add additional secrets to the Elasticsearch Pods that may be needed for sensitive files required for configuring Elasticsearch realms.
+The secret should be created by the user is the same namespace as the operator.
```

Comment:
```suggestion
The secret should be created by the user in the same namespace as the operator.
```

Is that true though? I thought StackConfigPolicies allow two modes: either in the namespace of the operator to configure any stack in the k8s cluster. Or in a specific namespace which restricts the policy to configure only resources in that namespace.

Reply:
@pebrc actually I messed up, in the code we expect it to be in the same namespace as the policy 
https://github.com/elastic/cloud-on-k8s/blob/41786785bc462d0c8bcc25d969792c2b9f97182e/pkg/controller/stackconfigpolicy/elasticsearch_config_settings.go#L82

Is that an acceptable behavior ?

Reply:
Yes that's fine. My point was more that the secret does not need to be in the same namespace as the operator but in the same namespace as the policy.